packages/assistify-help-request/config.js

```diff
@@ -1,7 +1,7 @@
 const _createExpertsChannel = function() {
 	const expertsRoomName = RocketChat.settings.get('Assistify_Expert_Channel');
 
-	if (!RocketChat.models.Rooms.findByNameContaining(expertsRoomName).fetch()) {
+	if (RocketChat.models.Rooms.findByNameContaining(expertsRoomName).fetch().length === 0) {
```

Comment:
Are you sure `fetch()` will always return a result/collection that is `not undefined` or could that happen?

Reply:
Well, the documentation(https://docs.meteor.com/api/collections.html#Mongo-Cursor-fetch) reads it returns an array. Nothing else. I verified that if no documents match the selector, an empty array is returned. 
An Exception may occur (such as an exception may always occur), but that's ... well an exception

Reply:
@rukurz anything else on your mind?

Reply:
nop. every thing fine. Thanks for that detailed explanation. Still learning ;-)